dali/base/dadfs.cpp

```diff
@@ -585,6 +585,37 @@ class CFileSuperOwnerLock : protected CFileLockCompound
     {
         return CFileLockCompound::detach();
     }
+    bool initWithFileLock(const CDfsLogicalFileName &logicalName, IRemoteConnection *conn, unsigned timeout, const char *msg, CFileLock fcl, unsigned fclmode)
```

Comment:
> CFileLock fcl,

Passing in object, not reference.
Surprised that didn't cause error or warning, but I suppose default copy constructor let it through.